azerothcore/azerothcore-wotlk
# fix(Core/The Oculus) Drakos dragon keepers

## CHANGES PROPOSED:
- Make Dragon Keepers outside of the cages
- Fix the broken NPC Texts and gossips, now they are blizzlike
- Add a wait time when npcs leave the cages, as on blizz


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3519



## TESTS PERFORMED:
- Kill Drakos
- Wait for them to move out
- Relog, they are still out.
- Tested the chats with different essences
- Restarted the server and did not reset instance, the npcs are in their place.



## HOW TO TEST THE CHANGES:

- Go to Oculus
```
.tele nexus
.gm on
```
- Kill Drakos
- See if the Drakes get out of the cages ( in humanoid form )
- Relog and see if they are still out
- Try to use the gossips of the npcs
- Try to restart the server ( and do not reset instances ) and see if they are still in the correct place
- Try to reset the instance and see if they are in the cages again.

## KNOWN ISSUES AND TODO LIST:

- [x] NPC BELGARISTRASZ is still silent after he finishes his walk
- [x] The gossips are messed up for all of them.
- [x] NPC's don't spawn where they should after server restart and instance in progress ( DRAKOS KILLED )


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
